frontend_tests/node_tests/top_left_corner.js

```diff
@@ -11,62 +11,12 @@ mock_esm("../../static/js/resize", {
 });
 
 const {Filter} = zrequire("../js/filter");
-const people = zrequire("people");
-const pm_list = zrequire("pm_list");
 const top_left_corner = zrequire("top_left_corner");
 
-run_test("narrowing", ({override_rewire}) => {
+run_test("narrowing", () => {
     // activating narrow
 
-    let pm_expanded;
-    let pm_closed;
-
-    override_rewire(pm_list, "close", () => {
-        pm_closed = true;
-    });
-    override_rewire(pm_list, "expand", () => {
-        pm_expanded = true;
-    });
-
-    assert.ok(!pm_expanded);
-    let filter = new Filter([{operator: "is", operand: "private"}]);
-    pm_list.handle_narrow_activated(filter);
-    assert.ok(pm_expanded);
-
-    const alice = {
-        email: "alice@example.com",
-        user_id: 1,
-        full_name: "Alice Smith",
-    };
-    const bob = {
-        email: "bob@example.com",
-        user_id: 2,
-        full_name: "Bob Patel",
-    };
-
-    people.add_active_user(alice);
-    people.add_active_user(bob);
-
-    pm_expanded = false;
-    filter = new Filter([{operator: "pm-with", operand: "alice@example.com"}]);
```

Comment:
Would it not be useful to add similar tests to check if the collapsed state of the PM list changes when we switch conversations, and to check if the list of items in the PM list is correct, etc? Why are you entirely deleting these tests?

Reply:
The PM list is no longer part of the top left corner. It makes sense to me to move tests relating to that to another file for PM tests. But yes lets keep these test cases. This functionality is actually broken right now, in that typing `is:private` in the search bar doesn't expand the PM list if it was collapsed. Can you add these test cases to `frontend_tests/node_tests/pm_list.js`?

Reply:
Ah interesting! So we don't want to expand it for individual conversations that makes sense. But what about when searching for "Private messages" in the search bar? Curious what Alya thinks :)

Reply:
I was expecting this not to interact with search... I think we can leave it as-is, and adjust if we realize we want to change it after seeing it in action.

Reply:
OK, thanks.

Reply:
Thanks for catching me up on this! Sounds good :)